VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -294,8 +285,7 @@ protected void onOnline() {
     @Override
     protected void onOffline() {
         super.onOffline();
-        showErrorMessage(getString(R.string.no_connectivity),
-                getString(R.string.network_not_connected), false);
+        showErrorMessage(getString(R.string.no_connectivity), getString(R.string.network_not_connected));
```

Comment:
While I agree with you on this, I think its best if I get approval from UX.
This seems like a requirement change.

On Mon, Aug 8, 2016 at 2:50 PM, Brian @ edX <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java
> <https://github.com/edx/edx-app-android/pull/794#discussion_r73931894>:
>
> > @@ -294,8 +285,7 @@ protected void onOnline() {
> >      @Override
> >      protected void onOffline() {
> >          super.onOffline();
> > -        showErrorMessage(getString(R.string.no_connectivity),
> > -                getString(R.string.network_not_connected), false);
> > +        showErrorMessage(getString(R.string.no_connectivity), getString(R.string.network_not_connected));
>
> Likewise, I don't think a dialog should popup if the user goes offline, it
> should only appear as a result of the user submitting the form. We can
> probably just delete this override and the onOnline override.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/edx/edx-app-android/pull/794/files/8fdfe5f3a21cec9d0bc9d4212121f0eb75e7b282#r73931894>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AR10T8YDmFakay3nVTbli8YkYjPGEj6Yks5qd3pegaJpZM4JfQbx>
> .
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX